### ISSUE TITLE
Add `xcinfo download` command

### ISSUE THREAD

**Issue by cntrump** [xcodereleases/xcinfo#18]
Just download xip, like this: `xcinfo download 12.5`

**Comment by trispo** [comment]
Thanks @cntrump for this suggestion. It will be added in the next release.

**Comment by ricobeck** [comment]
Resolved in 0.7.0

@cntrump please check and close this issue if it's working.

**Comment by cntrump** [comment]
> Resolved in 0.7.0
> 
> @cntrump please check and close this issue if it's working.

I try dowload `Xcode 5`: `xcinfo download 5`

```
Identifying
Found multiple possibilities for the requested version '5'.
1. 5.1.1 (5B1008)
2. 5.1 (5B130a)
3. 5.1 Beta 5 (5B103i)
4. 5.1 Beta 4 (5B90f)
5. 5.1 Beta 3 (5B71o)
6. 5.1 DP 2 (5B71f)
7. 5.1 DP 1 (5B45j)
8. 5.0.2 (5A3005a)
9. 5.0.1 (5A2053)
10. 5.0.1 GM Seed 1 (5A2039a)
11. 5.0 (5A1413)
12. 5.0 GM Seed 1 (5A1412)
13. 5.0 DP 6 (5A11386k)
14. 5.0 DP 5 (5A11365y)
15. 5.0 DP 4 (5A11365j)
16. 5.0 DP 3 (5A11344j)
17. 5.0 DP 2 (5A11315e)
18. 5.0 DP 1 (5A11314m)
Please choose the version you want to install: 11
Starting installation.

Sign in to Apple Developer
[7/7] Requesting download auth info

Downloading
Source: https://download.developer.apple.com/Developer_Tools/xcode_5/xcode_5.dmg (311 bytes)
[████████████████████] 100.0%, remaining: Zero KB, speed: -1 byte/s
Successfully downloaded to: /var/folders/30/2_r1y6zn4t721c1jx2_hydg00000gn/T/xcode_5.dmg
```
